p/themes/Ansum/ansum.css

```diff
@@ -1213,21 +1211,21 @@ main.prompt {
 		padding: 0.5rem 1rem;
 	}
 	.aside .toggle_aside,
-	#overlay .close,
-	.dropdown-menu .toggle_aside,
-	#slider .toggle_aside {
+#overlay .close,
+.dropdown-menu .toggle_aside,
+#slider .toggle_aside {
```

Comment:
Some indentation changes seem to have been accidentally included.

Reply:
fixed. thanks